### DISCUSSION TITLE
How to add a image on consent_modal

### DISCUSSION THREAD

**Discussion by amitsarkerr** [orestbida/cookieconsent#479] [Q&A] [answered]
Hello,

I want to show website logo on "consent_modal". I can not get any API to add image. Could you please give me a solution for this?

Thanks in advance.

**Comment by orestbida** [comment, marked as answer]
Hi @amitsarker89,

all fields accept `HTML` markup:

Example:

```js
title: '<img src="logo.png" alt="Logo" style="height: 35px;margin-top: 5px;">'
```

**Reply by amitsarkerr** [reply]
Thank you.
It works 🙂
